recipes_source/recipes/loading_data_recipe.py

```diff
@@ -1,32 +1,33 @@
 """
-Loading data in PyTorch
+PyTorch에서 데이터 불러오기
 =======================
-PyTorch features extensive neural network building blocks with a simple,
-intuitive, and stable API. PyTorch includes packages to prepare and load
-common datasets for your model.
+PyTorch는 인공신경망을 만드는데 필요한 다양한 기본 요소를 제공하고 있고, 간단하고, 직관적이고,
+안정적인 API로 쉽게 이용할 수 있습니다. PyTorch에는 공용 데이터셋을 쉽게 사용할
+수 있도록 도와주는 패키지를 포함하고 있습니다.
 
-Introduction
+소개
 ------------
-At the heart of PyTorch data loading utility is the
+PyTorch 데이터 불러오기 기능의 핵심에는
 `torch.utils.data.DataLoader <https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader>`__
-class. It represents a Python iterable over a dataset. Libraries in
-PyTorch offer built-in high-quality datasets for you to use in
-`torch.utils.data.Dataset <https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset>`__.
-These datasets are currently available in:
+클래스가 있습니다. 데어터를 파이썬 iterable로써 접근할 수 있게 해주는 클래스입니다.
```

Comment:
```suggestion
클래스입니다. 데어터를 파이썬 iterable로써 접근할 수 있게 해주는 클래스입니다.
```
이렇게 하는 게 조금 더 자연스러운 것 같아 제안해봅니다.

Reply:
데어터 -> 데이터 typo도 확인 부탁드립니다 :)